### PR TITLE
touying-simpl-hkustgz:0.1.0

### PR DESCRIPTION
**Pull request by exAClior**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: touying template for the Hong Kong University of Science and Technology(Guangzhou) 

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

**Comment by laurmaedje** [comment]
Packages named `something-*` where `something` is already an existing package are reserved for the owner of the original package. So in this case, @OrangeX4 would need to say whether they want to allow this or not.

File: packages/preview/touying-hkustgz/0.1.0/examples/main.typ [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+// appendix by freezing last-slide-number
+#let s = (s.methods.appendix)(self: s)
+#let (slide, empty-slide) = utils.slides(s)
+
+== 附注
+
+#slide[
+  - 您可以使用：
+    ```sh
+    typst init @preview/touying-buaa
```
Comment (OrangeX4) [inline packages/preview/touying-hkustgz/0.1.0/examples/main.typ:256]: Maybe it's the wrong package name.

File: packages/preview/touying-hkustgz/0.1.0/examples/main.typ [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+    ```
+    来创建基于本模板的演示文稿项目。
+
+  - 本模板仓库位于 #link("https://github.com/Coekjan/touying-buaa")，欢迎关注与贡献。
```
Comment (OrangeX4) [inline packages/preview/touying-hkustgz/0.1.0/examples/main.typ:260]: Same.

**Comment by OrangeX4** [comment]
I think using a "touying" prefix is feasible, which can help other users discover the theme of "touying". However, perhaps a more creative name could be used, such as "touying-night-hkustgz". But most of the current code is based on [touying-buaa](https://github.com/Coekjan/touying-buaa), so it might also be necessary to inform @Coekjan.

**Comment by laurmaedje** [comment]
> But most of the current code is based on [touying-buaa](https://github.com/Coekjan/touying-buaa), so it might also be necessary to inform @Coekjan.

Yeah, that one unfortunately slipped through because we didn't communicate the policy well enough internally.

**Comment by OrangeX4** [comment]
I think the code of touying-buaa is pretty good, and it is ok with the name touying-buaa. It's just that we can follow more creative naming principles for later packages, such as touying-hkustgz that only make minor changes to the touying-buaa code.

**Comment by Coekjan** [comment]
> But most of the current code is based on [touying-buaa](https://github.com/Coekjan/touying-buaa), so it might also be necessary to inform @Coekjan.

Would it be better if I add some notes in my repo like "If you want to create your template based on this repository, you should ..."?

**Comment by OrangeX4** [comment]
> Would it be better if I add some notes in my repo like "If you want to create your template based on this repository, you should ..."?

I think it's up to you. If you don't mind others distributing it in any form (with MIT License), just keep it that way.

File: packages/preview/touying-hkustgz/0.1.0/examples/main.typ (hunk 1 above)
Comment (exAClior) [reply packages/preview/touying-hkustgz/0.1.0/examples/main.typ]: Thanks for pointing out, I changed it.

File: packages/preview/touying-hkustgz/0.1.0/examples/main.typ (hunk 2 above)
Comment (exAClior) [reply packages/preview/touying-hkustgz/0.1.0/examples/main.typ]: Here too.

**Comment by exAClior** [comment]
> I think the code of touying-buaa is pretty good, and it is ok with the name touying-buaa. It's just that we can follow more creative naming principles for later packages, such as touying-hkustgz that only make minor changes to the touying-buaa code.

Sure, I have changed the naming to something more non-canonical.

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**